ui/src/components/mapbox/MapboxMap.tsx

```diff
@@ -59,13 +60,17 @@ const useMapboxMap = ({
       mapboxLibrary.accessToken = mapboxData.mapboxToken
 
     map.current = new mapboxLibrary.Map({
+      zoom: 6, // zoom must be initialized to 6 or more, or the language of map cannot be set.
+      center: [-77.0259, 38.901], // Initial to the land.
```

Comment:
Why did you hard-code these coordinates in particular?

Reply:
I set the map center, because if it isn't set, the center will be in the ocean by default. Then at zoom 6, the clients will see nothing on the map, and maybe they will think the map is broken. The best way is choosing the coordinate of the client's nation by default, but I don't find the way to get this information. So I have to hard-code a random coordinate in the land.

Reply:
Is it maybe appropriate to use the default [0, 0] coordinate and then have the default zoom be zoomed out to the max value so the entire world is visible and the user can zoom to their desired place?

Reply:
I set zoom to 6 because of a bug of mapbox-gl-language. [https://github.com/mapbox/mapbox-gl-language/issues/60](url)
In fact, at beginning I don't set either the zoom or the center, but just only call MapboxLanguage({defaultLanguage: xxx}). Then I found the map language is still English. I check the codes several times but have no idea. Fortunately, at last I found the BUG of mapbox-gl-language and raise the issue above. But until now, this bug is still here.
So now we have 2 choices:
1. Waiting mapbox-gl-language fix this bug, and don't set either the zoom or the center. But before they fix it, the map language will always be English.
2. Setting the zoom and center to provide multi-language map, waiting them fix this bug, and then delete the setting of zoom and center.
I want to fix this bug by myself, but I'm still learning JavaScript now.

Reply:
With the beginner’s luck, I found another way can change the map language without setting zoom to more than 6. If delete the setting of map projection in ui\src\Pages\PlacesPage\PlacesPage.tsx like this,
```
  const { mapContainer, mapboxMap, mapboxToken } = useMapboxMap({
    configureMapbox: configureMapbox({ mapboxData, dispatchMarkerMedia }),
    mapboxOptions: {
      //projection: {
      //  name: 'globe',
      //},
    },
  })
```
the map language can be changed too.
I still don't know why the map language is related to the zoom or projection. And I have a look to the code of mapbox-gl-language but found nothing about zoom or projection. 
I think may the map projection is important to the plane and ship, but not for a photo gallery :-). So, if you agree, I'll revert this commit, delete the setting of zoom and center, delete the setting of projection, and commit again.

Reply:
I move this commit to a new pull request #797, and this one will be closed.